eslint.config.js

```diff
@@ -3,26 +3,31 @@ import eslint from '@eslint/js';
 import tseslint from 'typescript-eslint';
 
 export default tseslint.config(
-	eslint.configs.recommended,
-	...tseslint.configs.strictTypeChecked,
-	{
-		languageOptions: {
-			globals: {
-				...globals.browser,
-				...globals.node,
-			},
-			parserOptions: {
-				project: true,
-				tsconfigRootDir: import.meta.dirname,
-			},
-		},
-	},
-	{
-		ignores: [
-			'**/coverage/**',
-			'**/dist/**',
-			'**/node_modules/**',
-			'**/__snapshots__/**',
-		],
-	},
+  eslint.configs.recommended,
+  ...tseslint.configs.strictTypeChecked,
+  {
+    languageOptions: {
+      globals: {
+        ...globals.browser,
+        ...globals.node,
+      },
+      parserOptions: {
+        project: true,
+        tsconfigRootDir: import.meta.dirname,
+      },
+    },
+  },
+  {
+    ignores: [
+      '**/coverage/**',
+      '**/dist/**',
+      '**/node_modules/**',
+      '**/__snapshots__/**',
+    ],
+  },
+  {
+    rules:{
+      'no-console':['error':{allow:['warn']}]
```

Comment:
```suggestion
      'no-console':['error':{allow:['warn']}]
```

let's disallow any console usage
and please fix the indentation to match the original